### PR TITLE
Unreal engine marketplace plugin changes

### PR DESCRIPTION
**Pull request by sanchitgarg**

### CODE REVIEW

File: targets/UnrealMarketplacePlugin/source/PlayFab/Source/PlayFabCommon/Private/PlayFabCommonSettings.cpp.ejs [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<%- copyright %>
+
+#include "PlayFabCommonSettings.h"
+
+namespace PlayFabCommon
+{
+    const FString PlayFabCommonSettings::sdkVersion = "<%- sdkVersion %>";
```
Comment (sanchitgarg) [inline targets/UnrealMarketplacePlugin/source/PlayFab/Source/PlayFabCommon/Private/PlayFabCommonSettings.cpp.ejs]: [ASK] These do not get populated correctly. They just show up as empty. Any idea?
Have already added unrealmarketplaceplugin to the SDKManualNotes.json file in the API_SPECS repo in my local branch.

File: targets/UnrealMarketplacePlugin/source/PlayFab/Source/PlayFabCpp/Private/PlayFab.cpp.ejs [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    // create the API
+    ClientAPI = MakeShareable(new PlayFab::UPlayFabClientAPI());
+    AuthenticationAPI = MakeShareable(new PlayFab::UPlayFabAuthenticationAPI());
+    CloudScriptAPI = MakeShareable(new PlayFab::UPlayFabCloudScriptAPI());
+    DataAPI = MakeShareable(new PlayFab::UPlayFabDataAPI());
+    EventsAPI = MakeShareable(new PlayFab::UPlayFabEventsAPI());
+    GroupsAPI = MakeShareable(new PlayFab::UPlayFabGroupsAPI());
+    ProfilesAPI = MakeShareable(new PlayFab::UPlayFabProfilesAPI());
+
+#if WITH_SERVER_CODE
```
Comment (pgilmorepf) [inline targets/UnrealMarketplacePlugin/source/PlayFab/Source/PlayFabCpp/Private/PlayFab.cpp.ejs]: This section is not properly templatized.
Check this out:
https://github.com/PlayFab/SDKGenerator/blob/33bfebcc79bafc5f50e511b40fe38b7bef0f0a0a/targets/cpp-unreal/PluginSource/Source/PlayFab/Private/PlayFab.cpp.ejs#L23

File: SDKBuildScripts/ue4_mkpl_build.bat [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+pushd ..\%destPath%
+del /S *.h
+del /S *.cpp
+del /S *.cs
+popd
+
+cd %~dp0
+pushd ..
+if [%1] == [] (
+rem === BUILDING UnrealCppSdk ===
```
Comment (pgilmorepf) [inline SDKBuildScripts/ue4_mkpl_build.bat]: UnrealCppSdk -> %repoName%

File: SDKBuildScripts/ue4_mkpl_build.bat [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+node generate.js %repoName%=%destPath% -apiSpecGitUrl -flags nonnullable
+) else (
+rem === BUILDING UnrealCppSdk with params %* ===
```
Comment (pgilmorepf) [inline SDKBuildScripts/ue4_mkpl_build.bat]: UnrealCppSdk -> %repoName%

File: targets/UnrealMarketplacePlugin/makeCommon.js [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+var maxEnumSize = 255;
+
+exports.makeCommonCombinedAPI = function (apis, copyright, sourceDir, apiOutputDir) {
+    var locals = {
+        apis: apis,
+        buildIdentifier: exports.buildIdentifier,
+        copyright: copyright,
+        errorList: apis[0].errorList,
+        errors: apis[0].errors,
+        friendlyName: "PlayFab Cpp Sdk",
```
Comment (pgilmorepf) [inline targets/UnrealMarketplacePlugin/makeCommon.js]: Wrong friendly name.

File: targets/UnrealMarketplacePlugin/source/PlayFab/PlayFab.uplugin.ejs [hunk 6]
```diff
@@ -0,0 +1,34 @@
+{
+    "FileVersion": 3,
+    "FriendlyName": "PlayFab Cpp Sdk",
```
Comment (pgilmorepf) [inline targets/UnrealMarketplacePlugin/source/PlayFab/PlayFab.uplugin.ejs]: Wrong friendly name.

File: targets/UnrealMarketplacePlugin/source/PlayFab/Source/PlayFabCpp/Private/PlayFab.cpp.ejs (hunk 2 above)
Comment (sanchitgarg) [reply targets/UnrealMarketplacePlugin/source/PlayFab/Source/PlayFabCpp/Private/PlayFab.cpp.ejs]: fixed

File: targets/UnrealMarketplacePlugin/source/PlayFab/Source/PlayFabCommon/Private/PlayFabCommonSettings.cpp.ejs (hunk 1 above)
Comment (sanchitgarg) [reply targets/UnrealMarketplacePlugin/source/PlayFab/Source/PlayFabCommon/Private/PlayFabCommonSettings.cpp.ejs]: fixed

File: targets/UnrealMarketplacePlugin/source/PlayFab/PlayFab.uplugin.ejs (hunk 6 above)
Comment (sanchitgarg) [reply targets/UnrealMarketplacePlugin/source/PlayFab/PlayFab.uplugin.ejs]: fixed

File: targets/UnrealMarketplacePlugin/makeCommon.js (hunk 5 above)
Comment (sanchitgarg) [reply targets/UnrealMarketplacePlugin/makeCommon.js]: fixed

File: SDKBuildScripts/ue4_mkpl_build.bat (hunk 4 above)
Comment (sanchitgarg) [reply SDKBuildScripts/ue4_mkpl_build.bat]: fixed

File: SDKBuildScripts/ue4_mkpl_build.bat (hunk 3 above)
Comment (sanchitgarg) [reply SDKBuildScripts/ue4_mkpl_build.bat]: fixed

**Pull request merged**